### PR TITLE
allow to set a custom cloud6/7 iso path and name

### PR DESCRIPTION
**Pull request by tpatzig**

### CODE REVIEW

File: scripts/lib/mkcloud-common.sh [hunk 1]
```diff
@@ -13,6 +13,7 @@
 distsuseip=$(dig -t A +short $distsuse)
 : ${susedownload:=download.nue.suse.com}
 : ${smturl:="http://$susedownload/update/build.suse.de"}
+
```
Comment (bmwiedemann) [inline scripts/lib/mkcloud-common.sh]: leftover from earlier version?

**Comment by bmwiedemann** [comment]
scripts/lib/qa_crowbarsetup-help.sh would be the right place to document these variables, as they are not used on the mkcloud-host.

**Comment by gonzolino** [comment]
Added documentation to scripts/lib/qa_crowbarsetup-help.sh and removed the empty line.

**Comment by tpatzig** [comment]
we added the want_cloud{6,7}_iso* variables to cloudsource GM{6,7} as well; and we moved the defaults back to `function onadmin_set_source_variables`

File: scripts/qa_crowbarsetup.sh [hunk 2]
```diff
@@ -1218,16 +1218,16 @@ function onadmin_set_source_variables
             CLOUDLOCALREPOS="SUSE-Cloud-5-devel"
         ;;
         develcloud6)
-            CLOUDSLE12DISTPATH=/ibs/Devel:/Cloud:/6/images/iso
+            CLOUDSLE12DISTPATH=${want_cloud6_iso_path:='/ibs/Devel:/Cloud:/6/images/iso'}
```
Comment (jdsn) [inline scripts/qa_crowbarsetup.sh]: This feels strange, as the default here is different than the default in line 1262 - this indicates that we should rather restructure how we do the mapping here (eg. read it with external tool from a yaml file that holds the mappings).
But as long as SAP only needs one variable here, this should not be a problem.

**Pull request merged**